USER_MANUAL.md

```diff
@@ -300,7 +306,81 @@ Note that programs are interpreted as a flat list of parameters that are incorpo
 
 **Notes of caution:** Improper versioning will prevent simulator from executing.
 
-_TODO_
+### &lt;n_processes&gt;
+
+**Data type:** Integer (Numeric)
+
+**Default input:** 6
+
+**Description:** The maximum number of parallel tasks/processes that the simulator is allowed to use at a given time.
```

Comment:
Can you describe in more detail what specifying more or less processes will do as far as a user is concerned?